consultation_analyser/consultations/models.py

```diff
@@ -61,9 +61,13 @@ class Theme(UUIDPrimaryKeyModel, TimeStampedModel):
     label = models.CharField(max_length=256, blank=True)
     summary = models.TextField(blank=True)
     keywords = models.JSONField(default=list)
+    # Duplicates info in Answer model, but needed for uniqueness constraint.
```

Comment:
whilst im not 100% that this would work (or even be worth it) you could do something like...


```python
constraints = [
            models.UniqueConstraint(fields=["summary", "label", "keywords", "answer_set__question"], name="unique_up_to_question"),
        ]
```

Reply:
Yeah, it doesn't work with related fields (which would have made things much easier).